### ISSUE TITLE
[Package Issue]: Bambulab.Bambustudio 01.09.01.67

### ISSUE THREAD

**Issue by intdel** [microsoft/winget-pkgs#155221]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

Old version of Bambulab.Bambustudio 01.09.01.66 was 404 when downloading. Bambulab seems to have replaced the binary with 01.09.01.67.

### Steps to reproduce

Try to install Bambustudio.

### Actual behavior

```
(1/1) Found Bambu Studio [Bambulab.Bambustudio] Version 01.09.01.66
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/bambulab/BambuStudio/releases/download/v01.09.01.66/Bambu_Studio_win_public-v01.09.01.66-20240511203248.exe
An unexpected error occurred while executing the command:
Download request status is not success.
0x80190194 : Not found (404).
```

### Expected behavior

Download works

### Environment

```raw
Windows Package Manager v1.7.11261
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22631.3593
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.22.11261.0
```

**Comment by intdel** [comment]
Fixed in #155219

**Comment by stephengillie** [comment]
> Fixed in #155219
